### PR TITLE
Wrong task script injected in iframe HTML (close #534)

### LINKED ISSUE #534: Wrong task script injected in iframe HTML
A regular `task.js` instead of `iframe-task.js` is injected into iframe page, when using a `form` with `target` attribute that is initially empty, but is later specified through JS.

Example:
Test page:
```html
<html>
	<head>
		<title>Test iframe</title>
	</head>
	<body>
	    <form action="iframe.html">
                <input type="submit" />
            </form>
            <iframe name="iframe">
            </iframe>
            <script type="text/javascript">
                var form    = document.getElementsByTagName('form')[0];
                var button = document.getElementsByTagName('input')[0];
            
                button.addEventListener('click', function () {
                    form.target = 'iframe';
                });            

            </script>
	</body>
</html>
```
Content of `iframe.html` doesn't matter, it may be even not HTML, just a plain text. Start `playground` and load the page. Click on submit button, and check the head of iframe's document. It will contain `<script ... src="task.js">` instead of `<script ... src="iframe_task.js">`.

### PR DESCRIPTION
**Pull request by churkin**
\cc @LavrovArtem @kirovboris

### CODE REVIEW

**Comment by kirovboris** [comment]
lgtm

File: src/client/sandbox/node/element.js [hunk 1]
```diff
@@ -29,6 +29,26 @@ export default class ElementSandbox extends SandboxBase {
         this.BEFORE_FORM_SUBMIT = 'hammerhead|event|before-form-submit';
     }
 
+    static _onTargetChanged (el, newTarget) {
+        var urlAttr        = domUtils.getTagName(el) === 'form' ? 'action' : 'href';
```
Comment (LavrovArtem) [inline src/client/sandbox/node/element.js]:
```javascript
domUtils.isFormElement(el)
```

**Comment by LavrovArtem** [comment]
\r-

File: src/client/sandbox/node/element.js (hunk 1 above)
Comment (churkin) [inline src/client/sandbox/node/element.js]: This unnecessarily in this case

**Comment by churkin** [comment]
FPR

**Comment by LavrovArtem** [comment]
lgtm

**Pull request merged**